### ISSUE TITLE
Include alt text on Core Team images 

### ISSUE THREAD

**Issue by maneesha** [carpentries/carpentries.org-archive#1552]
Per suggestion from @brynnelliott  in slack, we should have alt text on the images on our Core Team page (https://carpentries.org/team/).

**Comment by maneesha** [comment]
Re-opening this because #1553 fixed adding standard info to the alt text.  We may want to give Core Team members the option of adding more descriptive text to their image.  I'll come back to this next year.

**Comment by maneesha** [comment]
Closed with #1647
I will review how to add your own alt text at an upcoming Core Team meeting.

**Comment by maneesha** [comment]
Marking this as complete as this has been reviewed with all Core Team members, we all have new headshots, and we all have an optional task to update our alt text.
